### PR TITLE
Add seed widget

### PR DESCRIPTION
**Pull request by chakravarthi589**

### CODE REVIEW

File: src/Java/Widgets/SeedWidget.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Description
+
+This example demonstrates how to use vtkSeedWidget, which generates (seeds) points to be placed in the scene in the locations where the user clicks.
+
+The points can then be used for operations like connectivity, segmentation, and region growing. For an example using a custom callback where such operations can be assembled, see SeedWidgetWithCustomCallback.
```
Comment (ajpmaclean) [inline src/Java/Widgets/SeedWidget.md]: Regarding SeedWidgetWithCustomCallback are you referring to the Cxx example as the Java example does not exist?
If so this needs to be a link, look at the markdown file in e.g. [Cube.cxx](https://lorensen.github.io/VTKExamples/site/Cxx/GeometricObjects/Cube/) for how to do this.

File: src/Java.md [hunk 2]
```diff
@@ -220,4 +220,5 @@ It would be appreciated if there are any Java VTK experts who could convert any
 [AngleWidget2D](/Java/Widgets/AngleWidget2D) | vtkAngleWidget2D | AngleWidget2D Demonstration
 [BalloonWidget](/Java/Widgets/BalloonWidget) |  vtkBalloonWidget | The balloon text describes each object when you hover it.
 [PlaneWidget](/Java/Widgets/PlaneWidget) | vtkPlaneWidget | A finite (bounded) plane that can be interactively placed in a scene.
+[SeedWidget](/Java/Widgets/SeedWidget) | vtkSeedWidget | Places multiple seed points.
```
Comment (ajpmaclean) [inline src/Java.md]: Change "Places multiple seed points. " to "Places multiple seed points, where a user moves the cursor to and clicks it. "

Comment (chakravarthi589) [reply src/Java.md]: I have updated.

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**